Framework/include/QualityControl/HistoProducer.h

```diff
@@ -0,0 +1,39 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   HistoProducer.h
+/// \author Barthelemy von Haller
+///
+
+#ifndef QUALITYCONTROL_HISTOPRODUCER_H
+#define QUALITYCONTROL_HISTOPRODUCER_H
+
+#include <Framework/DataProcessorSpec.h>
+
+namespace o2::quality_control::core
+{
+
+/// \brief Returns a random data producer specification which publishes on {"TST", "RAWDATA", <index>}
+///
+/// \param myParam  The value the producer should produce.
+/// \return         A fixed number producer specification
+framework::DataProcessorSpec getHistoProducerSpec(size_t myParam);
+
+/// \brief Returns an algorithm generating random messages
+///
+/// \param output   Origin, Description and SubSpecification of data to be produced
+/// \param  myParam  The value the producer should produce.
```

Comment:
The description of `myParam` is unclear in this context (same in the function above)